docker-compose.yml

```diff
@@ -2,9 +2,12 @@ version: '2.4'
 
 x-web-defaults: &web-defaults
   restart: unless-stopped
+  image: ghcr.io/apexarray/obico-server/web:dev-django-4.2
```

Comment:
Having both "image" and "build" here are quite confusing. Is the intention to run the container straight off of the image, or build it?

Reply:
reverted in [fad44bb](https://github.com/TheSpaghettiDetective/obico-server/pull/829/commits/fad44bb4f22e44437cf37544be1198f1f35933ec)